queries/places.jinja2

```diff
@@ -70,9 +74,13 @@ SELECT
     NULL AS population,
     NULL AS capital,
     NULL AS state_capital,
+    n_photos,
```

Comment:
`n` is "number of", usually, and that seems to fit in this situation. If we're not planning to expose this publicly (except perhaps during development) then perhaps we should put an `mz_` prefix on it?